hs_core/views/__init__.py

```diff
@@ -578,7 +576,6 @@ def update_metadata_element(request, shortkey, element_name, element_id, *args,
     return HttpResponseRedirect(request.META['HTTP_REFERER'])
 
 
-@swagger_auto_schema(method='get', auto_schema=None)
```

Comment:
why'd you remove this?  It tells swagger not to include this in hsapi.

Reply:
Likely an accident. I'll look at which commit/feature branch it came from to make sure it's caught in all places.

Reply:
Scott, doesn't appear as though I removed this. Looks like the line was present in `c83e8655` and then missing after a dev merge `013171a`. I'm not going to speculate what could have happened. I'm just adding it back and marking this as resolved.

Reply:
Fixed in 8c3612ea5